website/templates/profile/account.mako

```diff
@@ -66,12 +65,15 @@
                                 <tr>
                                     <td colspan="3">
                                         <form data-bind="submit: addEmail">
+                                        <p>To merge an existing OSF account with this one or to be able to log in under multiple email addresses, add your alternate email address in the field below.</p>
```

Comment:
Honestly? As a user, if I try to register an email address already is use, I'd expect an error message. (not a magic link a la [#OSF-4542]) The merge behavior was a little surprising to me at first discovery, and very powerful relative to the amount of fanfare.

There was some discussion of text on a previous round- see the associated JIRA issue.